diff -urN busybox-1.30.1.orig/networking/udhcp/common.c busybox-1.30.1/networking/udhcp/common.c
--- busybox-1.30.1.orig/networking/udhcp/common.c	2019-02-14 14:31:15.000000000 +0100
+++ busybox-1.30.1/networking/udhcp/common.c	2019-04-05 14:24:04.875216049 +0200
@@ -67,6 +67,9 @@
 	{ OPTION_STRING                           , 0xd2 }, /* DHCP_PXE_PATH_PREFIX */
 	{ OPTION_U32                              , 0xd3 }, /* DHCP_REBOOT_TIME   */
 	{ OPTION_6RD                              , 0xd4 }, /* DHCP_6RD           */
+	{ OPTION_STRING               | OPTION_REQ, 0xe1 }, /* DHCP_SI_SERVER */
+	{ OPTION_STRING               | OPTION_REQ, 0xe2 }, /* DHCP_SI_MONITOR */
+	{ OPTION_STRING               | OPTION_REQ, 0xe3 }, /* DHCP_SI_SYSLOG */
 	{ OPTION_STATIC_ROUTES | OPTION_LIST      , 0xf9 }, /* DHCP_MS_STATIC_ROUTES */
 	{ OPTION_STRING                           , 0xfc }, /* DHCP_WPAD          */
 
@@ -136,6 +139,9 @@
 	"pxepathprefix" "\0"    /* DHCP_PXE_PATH_PREFIX */
 	"reboottime" "\0"       /* DHCP_REBOOT_TIME     */
 	"ip6rd" "\0"            /* DHCP_6RD             */
+	"siserver" "\0"			/* DHCP_SI_SERVER  */
+	"simonitor" "\0"		/* DHCP_SI_MONITOR  */
+	"sisyslog" "\0"			/* DHCP_SI_SYSLOG  */
 	"msstaticroutes" "\0"   /* DHCP_MS_STATIC_ROUTES*/
 	"wpad" "\0"             /* DHCP_WPAD            */
 	;
diff -urN busybox-1.30.1.orig/networking/udhcp/common.h busybox-1.30.1/networking/udhcp/common.h
--- busybox-1.30.1.orig/networking/udhcp/common.h	2019-02-14 14:31:15.000000000 +0100
+++ busybox-1.30.1/networking/udhcp/common.h	2019-04-05 14:22:05.835211791 +0200
@@ -157,6 +157,9 @@
 //#define DHCP_PXE_CONF_FILE    0xd1 /* 209: RFC 5071 Configuration File */
 //#define DHCP_PXE_PATH_PREFIX  0xd2 /* 210: RFC 5071 Configuration File */
 //#define DHCP_REBOOT_TIME      0xd3 /* 211: RFC 5071 Reboot time */
+//#define DHCP_SI_SERVER        0xe1 /* Simple Imager server; "<hostname>[:<port>][!<protocol>][!<protocol>]" */
+//#define DHCP_SI_MONITOR       0xe2 /* Simple Imager monitor; "{<hostname>[:<port>]|<url>}[!<protocol>]" */
+//#define DHCP_SI_SYSLOG        0xe3 /* Syslog (RFC 3164) server; "<hostname>[:<port>]" */
 //#define DHCP_MS_STATIC_ROUTES 0xf9 /* 249: Microsoft's pre-RFC 3442 code for 0x79? */
 //#define DHCP_WPAD             0xfc /* 252: MSIE's Web Proxy Autodiscovery Protocol */
 #define DHCP_END                0xff /* 255: */

