--- pciutils-3.6.2.orig/Makefile	2019-04-08 11:14:07.539452952 +0200
+++ pciutils-3.6.2/Makefile	2019-04-08 11:18:13.363440043 +0200
@@ -13,19 +13,19 @@
 CROSS_COMPILE=
 
 # Support for compressed pci.ids (yes/no, default: detect)
-ZLIB=
+ZLIB=yes
 
 # Support for resolving ID's by DNS (yes/no, default: detect)
-DNS=
+DNS=no
 
 # Build libpci as a shared library (yes/no; or local for testing; requires GCC)
 SHARED=no
 
 # Use libkmod to resolve kernel modules on Linux (yes/no, default: detect)
-LIBKMOD=
+LIBKMOD=yes
 
 # Use libudev to resolve device names using hwdb on Linux (yes/no, default: detect)
-HWDB=
+HWDB=yes
 
 # ABI version suffix in the name of the shared library
 # (as we use proper symbol versioning, this seldom needs changing)

